catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -263,15 +272,25 @@ public void configure() throws Exception {
                 // Configure the camel route to ignore changing file (larger files that are in the process of being copied)
                 // Set the readLockTimeout to continuously poll the directory so long as the directory monitor exists
                 // Set the readLockCheckInterval to check every 5 seconds
-                String inbox = "file:" + monitoredDirectory + "?moveFailed=.errors&readLock=changed&readLockTimeout=0&readLockCheckInterval=5000";
-                if (copyIngestedFiles) {
-                    inbox += "&move=.ingested";
-                } else {
-                    inbox += "&delete=true";
+                StringBuilder inbox = new StringBuilder("file:" + monitoredDirectory
```

Comment:
If you have a StringBuilder already, it doesn't make much sense to then append a String to initialize it. Use .append() for these or change it back to a String and let the compiler optimize it (it will create a StringBuilder anyways).